### ISSUE TITLE
Connection to clusters with external kubeconfig files lost after upgrade to 5.0.0

### ISSUE THREAD

**Issue by Kolossi** [lensapp/lens#3221]
**Describe the bug**
Ability to connect to clusters through external kubeconfig files broken in 5.0.0

**To Reproduce**
1. Dialog displayed offering to upgrade to latest version 5.0.0.  No warning that accepting will lose all connectivity to clusters if using external kubeconfig files.
2. Click to upgrade.
3. After restart with 5.0.0 previous clusters are shown in "hotbar" but saying "unknown".  Clusters can be added but only with copy-paste content (and with warning that clusters will not be merged into .kubeconfig) and not by accessing existing files.
4. %AppData%/Roaming/Lens/lens-cluster-store.json still contains cluster setups with correct "kubeConfigPath", and the kubeconfig file still exists and with correct content.

**Expected behavior**
1. Support for multiple external kubeconfigs should be supported (I don't want or need cloud sync of single kubeconfig file).
2. Failing that at the very least a strong warning that connectivity will be lost should be displayed before upgrade when external kubeconfig files are configured.

**Environment (please complete the following information):**
- Lens Version: now 5.0.0
- OS: Windows
- Installation method (e.g. snap or AppImage in Linux): in-app upgrade from previous version.

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
info: ≡ƒôƒ Setting Lens as protocol client for lens://
info: ≡ƒôƒ Protocol client register succeeded Γ£à
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
n/a - previous configs worked fine in v4.x, current kubeconfigs are not even processed
```

**Additional context**
none, but need access to clusters via lens asap.

**Comment by Kolossi** [comment]
Digging further and comparing old (not working) and new entries created by copy-paste in `%AppData%/Roaming/Lens/lens-cluster-store.json` it seems the difference between the entries is minimal, and only that old entries have:

```
"kubeConfigPath": "C:\\Users\\{ME}\\.kube-lens\\k8s-pp-blue-sa.config"
```

New ones have:

```
"kubeConfigPath": "C:\\Users\\{ME}\\AppData\\Roaming\\Lens\\kubeconfigs\\60909c42-e3f4-4877-8d5d-e0c453d9e425"
```

So a guid rather than named filename and a different path.  Not sure that would stop it being loaded though, unless new entries have extra supporting records elsewhere?

If so, ofc the relevant entries should be created by migration on install of v5.0.0 imho.

**Comment by Kolossi** [comment]
Still no idea why current configs aren't working as they show up in sidebar, but if anyone else reads this, following [advice on slack](https://k8slens.slack.com/archives/C1U90NQN8/p1625045050141900), the solution is to add files (or directories) containing external kubeconfig files as a "Kubeconfig sync" under the "Kubernetes" tab of preferences in v5.x

Fix is apparently slated for v5.0.1
